src/safe-upgrade-tool/introduction.md

```diff
@@ -0,0 +1,16 @@
+---
+group: software-update-guide
+title: Introduction
+functional_areas:
+  - Upgrade
+---
+
+The Safe Upgrade Tool (SUT) is a command line (CLI) tool that checks a Magento instance against a specific version by analyzing all the non-Magento modules installed in it.
+
+The SUT identifies potential problems that must be fixed in your custom code before attempting to upgrade to a newer version of Magento.
+
+The tool returns a list of errors and warnings that you must address before upgrading to a new version of Magento.
+
+This tool is distributed as a composer package with every release of a Magento version. See [Developer]({{site.baseurl}}/safe-upgrade-tool/developer.html) topic for more information.
```

Comment:
As far as I know it has not been decided yet that we want to package the tool with EACH release of Magento, including all beta versions. This needs to be clarified with Sandra

Reply:
We should have a version of the tool for each released version. @alena-marchenko will be working on an automated process to generate the Magento API index with every new version we release.

Reply:
should we keep this paragraph as is then?